config/default.yaml

```diff
@@ -119,6 +119,15 @@ executor:
 
         # Launcher container tag to use
         launchVersion: stable
+    jenkins:
+      # Configuration of Jenkins
+      jenkins:
```

Comment:
That's also how the `executor-router` handles executors https://github.com/screwdriver-cd/executor-router